### ISSUE TITLE
Misuse of loss function in 0_basic.py tutorial?

### ISSUE THREAD

**Issue by guoxuxu** [thunlp/OpenPrompt#105]
Here is the basic flow that captures the loss computation in 0_basic.py tutorial. 
```
...
myverbalizer = ManualVerbalizer(tokenizer, num_classes=3, 
                        label_words=[["yes"], ["no"], ["maybe"]])
...
prompt_model = PromptForClassification(plm=plm,template=mytemplate, verbalizer=myverbalizer, freeze_plm=False)
...
loss_func = torch.nn.CrossEntropyLoss()
...
logits = prompt_model(inputs)
labels = inputs['label']
loss = loss_func(logits, labels)
```
The ```post_log_softmax``` of ```ManualVerbalizer``` defaults to True, which computes the log softmax on the output logits of the PLM. Therefore, ```logits = prompt_model(inputs)``` gives the log(softmax(x)) results. Shouldn't we use ```NLLLoss``` after that?  If we use ```CrossEntropyLoss```, it will compute softmax again. I don't understand...

**Comment by Achazwl** [comment]
This would result in the same loss and the same gradient. Of course, you can change this to NLLLoss.
